www/source/tutorials.html.slim

```diff
@@ -20,8 +20,21 @@ header.child-h.blue-gradient.margin-top-offset
   .columns.medium-12
     h3 Tutorials
 
-    hr.strict-left 
+    hr.strict-left
 
+    br
+    p.clear Learn more about InSpec and earn badges in these interactive tracks on <a href="https://learn.chef.io/">Learn Chef Rally</a>.
+    ul.tutorials--links.no-bullet
+      li
+        a.tutorials--link href="https://learn.chef.io/tracks/compliance-automation/"
+          i.focus &GreaterGreater;
+          |  Compliance Automation with InSpec
+          div  Start here. Downloadable training environments get you started with everything you need.
+      li
+        a.tutorials--link href="https://learn.chef.io/tracks/integrated-compliance/"
+          i.focus &GreaterGreater;
+          |  Integrated Compliance with Chef
+          div  Continue your journey here if you're interested in using Chef to correct potential compliance and security issues.
     br
     p.clear Huge thanks to Annie Hedgpeth for creating all of these fantastic tutorials.
```

Comment:
To make it clear which of the tutorials are Annie's, I suggest changing this line to:

"Huge thanks to Annie Hedgpeth for creating all of the following fantastic tutorials:"